index.html

```diff
@@ -284,6 +284,9 @@ <h2>Scope</h2>
 document as input, the WG will produce a NOTE at the end of the process that is
 a refined Use Cases document.
           </li>
+          <li>
+Recommend requirements for DID methods, including which operations must be supported, which must be described, and which are optional.
+          </li>
         </ol>
```

Comment:
Would also recommend that we include something like:

* Establishment of a deterministic mapping between DID method identifiers and the resolution process used to resolve that DID method.

Reply:
Added in 2aa2cedba17dba1377d7b36d687257d678851800.